client/components/TestReview/index.jsx

```diff
@@ -0,0 +1,270 @@
+                        )}
+                    </Ul>
+                </li>
+                <li>
+                    <strong>Commit:&nbsp;</strong>
+                    {testPlanVersion.gitMessage}
+                </li>
+            </ul>
+            <h2>List of Tests</h2>
+            <FilterButtonContainer>
+                <FilterButtons
+                    filterLabel="Filter tests by AT"
+                    filterOptions={filterOptions}
+                    activeFilter={activeFilter}
+                    onFilterChange={selectedFilter => {
+                        setActiveFilter(selectedFilter);
+                    }}
+                />
+            </FilterButtonContainer>
+            {filteredTests.map((test, index) => {
+                const isFirst = index === 0;
+
+                const atMode =
+                    test.atMode.substring(0, 1) +
+                    test.atMode.toLowerCase().substring(1);
+
+                const specifications =
+                    test.renderableContents[0].renderableContent.info.references.map(
+                        ({ refId, value }) => [refId, value]
+                    );
+
+                let filteredAts;
+                if (activeFilter === 'All ATs') {
+                    filteredAts = test.ats;
+                } else {
+                    filteredAts = test.ats.filter(
+                        at => at.name === activeFilter
+                    );
+                }
+
+                return (
+                    <Fragment key={test.id}>
+                        {isFirst ? null : <hr />}
+                        <h3>{`Test ${test.rowNumber}: ${test.title}`}</h3>
+                        <ul>
+                            <li>
+                                <strong>Mode:&nbsp;</strong>
+                                {atMode}
+                            </li>
+                            <li>
+                                <strong>Assistive technologies:&nbsp;</strong>
+                                {test.ats.map(at => at.name).join(', ')}
+                            </li>
+                            <li>
+                                <strong>Relevant specifications:&nbsp;</strong>
+                                {specifications.map(([title, link], index) => {
+                                    const isLast =
+                                        index === specifications.length - 1;
+                                    return (
+                                        <Fragment key={title}>
+                                            <a
+                                                href={link}
+                                                rel="noreferrer"
+                                                target="_blank"
+                                            >
+                                                {title}
+                                            </a>
+                                            {isLast ? '' : ', '}
```

Comment:
I understand that this is more visually compact by just making them into sibling <a>'s but the tradeoff is that the comma confuses VoiceOver (other ATs untested). I think it is worth the added scroll height to have a <ul> -> <li> -> <a> structure without commas here.

Reply:
Sure, good idea